### DISCUSSION TITLE
Highlighting functions inside functions!

### DISCUSSION THREAD

**Discussion by sorrowlol** [tiagovla/tokyodark.nvim#16] [Ideas]
Also I was wondering if it would be possible to highlight functions with the same green color inside functions?

Sorry for these, i am quite new to VIM and still learning! I tried to do it myself but couldn't get a working version

**Comment by tiagovla** [comment]
Sorry, I didn't get any github notifications. What language are you using?
![image](https://user-images.githubusercontent.com/30515389/158891987-66aeb3ed-c072-428f-ad0a-c980d5eb179b.png)
This is a python example, do you want builtin functions to also be green?

**Reply by sorrowlol** [reply]
no probs! yup the built int to also be green. I am using Ruby  :)

**Comment by tiagovla** [comment]
@sorrowlol this should do it
```vim
:hi! link TSFuncBuiltin TSFunction
```

**Reply by tiagovla** [reply]
The lua version if you are using nightly:
```lua
vim.api.nvim_set_hl(0, "TSFuncBuiltin", { link="TSFunction" })
```

**Reply by sorrowlol** [reply]
that did the trick. Thank you!
